Sources/ApolloCodegenLib/Frontend/JavaScript/src/__testUtils__/loading.ts

```diff
@@ -0,0 +1,72 @@
+import * as fs from 'fs';
+import { stripIndents } from "common-tags";
+
+import {
+  buildClientSchema,
+  Source,
+  concatAST,
+  parse,
+  DocumentNode,
+  GraphQLSchema,
+} from "graphql";
+
+export function loadSchema(schemaPath: string): GraphQLSchema {
+  if (!fs.existsSync(schemaPath)) {
+    throw new Error(`Cannot find GraphQL schema file: ${schemaPath}`);
+  }
+  const schemaData = require(schemaPath);
+
+  if (!schemaData.data && !schemaData.__schema) {
+    throw new Error(
+      "GraphQL schema file should contain a valid GraphQL introspection query result"
+    );
+  }
+  return buildClientSchema(schemaData.data ? schemaData.data : schemaData);
+}
+
+export function loadQueryDocuments(
+  inputPaths: string[],
+): DocumentNode[] {
+  const sources = inputPaths
+    .map(inputPath => {
+      if (fs.lstatSync(inputPath).isDirectory()) {
+        return null;
+      }
+
+      const body = fs.readFileSync(inputPath, "utf8");
+      if (!body) {
+        return null;
+      }
+
+      if (
+        inputPath.endsWith(".graphql") ||
+        inputPath.endsWith(".graphqls") ||
```

Comment:
It's pretty common to be able to differentiate an SDL file from other GraphQL documents, especially when using wildcards to select input files. I even did this for our own Star Wars API, because using `.graphql` would include the SDL as codegen input and would lead to errors.

Reply:
Like I said above, I've generally seen that saved as `.sdl`